src/api/routes.ts

```diff
@@ -227,6 +227,13 @@ export const buildPostItemLikeRoute = (id: UUID) => `${ITEMS_ROUTE}/${id}/like`;
 export const buildDeleteItemLikeRoute = (id: UUID) =>
   `${ITEMS_ROUTE}/likes/${id}`;
 
+export const VALIDATION_ROUTE = 'validations'
+export const buildGetValidationStatusRoute = (id: UUID) => `${ITEMS_ROUTE}/${VALIDATION_ROUTE}/status/${id}`;
+export const GET_VALIDATION_REVIEW_ROUTE = `${ITEMS_ROUTE}/${VALIDATION_ROUTE}/reviews`;
+export const GET_ALL_STATUS_ROUTE = `${ITEMS_ROUTE}/${VALIDATION_ROUTE}/status`;
```

Comment:
`GET_VALIDATION_STATUSES_ROUTE`? Maybe we want to change the endpoint for statuses as well?